.github/workflows/master-e2e.yaml

```diff
@@ -297,17 +297,12 @@ jobs:
           UPGRADE_TYPE: osImage
           VM_INDEX: 1
         run: cd tests && make e2e-upgrade-node
-      - name: Bootstrap node 2 with current build (use ISO)
+      - name: Bootstrap node 2 and 3 with current build (use ISO)
         if: inputs.test_type == 'cli'
         env:
           ISO_BOOT: true
           VM_INDEX: 2
```

Comment:
Is the `VM_INDEX: 2` still valid?

Reply:
Yes, `VM_INDEX` is the number where start (so node 2) and `VM_NUMBERS` is the total number of node to have at the end, so 3 here. ANd in fact maybe better to rename the variables to `NODE_*` instead of `VM_*`, if we have the test running on *real* HW later. But this can be done later too.